.bcr/source.template.json

```diff
@@ -1,5 +1,5 @@
 {
   "integrity": "",
   "strip_prefix": "{REPO}-{VERSION}",
-  "url": "https://github.com/{OWNER}/{REPO}/archive/refs/tags/{TAG}.tar.gz"
+  "url": "https://github.com/{OWNER}/{REPO}/releases/download/{VERSION}/${REPO}-${VERSION}.tar.gz"
```

Comment:
The current URL is https://github.com/bazel-contrib/rules_jvm/releases/download/v0.21.1/rules_jvm-v0.21.1.tar.gz, so I think you need to replace `VERSION` with `TAG`.

Reply:
Gotta love that the v-adding/stripping behaviour. Good catch, thanks!